lib/tuya.js

```diff
@@ -581,6 +581,26 @@ const dataPoints = {
     // Moes switch with optional neutral
     moesSwitchPowerOnBehavior: 14,
     moesSwitchIndicateLight: 15,
+    // X5H thermostat
+    x5hState: 1, // DONE
+    x5hMode: 2, // DONE // equals to moesHold
+    x5hWorkingStatus: 3, // DONE
+    x5hSound: 7, // DONE
+    x5hFrostProtection: 10, // DONE // equals to tvFrostDetection // test if works
+    x5hSetTemp: 16, // DONE // equals to moesHeatingSetpoint
+    x5hSetTempCeiling: 19, // DONE // max temperature // equals to moesMaxTemp
+    x5hCurrentTemp: 24, // DONE // equals to moesLocalTemp
+    x5hTempCorrection: 27, // DONE // equals to moesTempCalibration BUT CHECK IF WORKS OK
+    x5hWeeklyProcedure: 30, // MAYBE equals to moesSchedule
+    x5hWorkingDaySetting: 31, // DONE // equals to tvWorkingDay // thermostatWeekFormat
+    x5hFactoryReset: 39, // DONE
+    x5hChildLock: 40, // DONE // equals to moesChildLock
+    x5hSensorSelection: 43, // DONE // equals to moesSensor
+    x5hFaultAlarm: 45, // DONE
+    x5hTempDiff: 101, // DONE
+    x5hProtectionTempLimit: 102, // DONE
+    x5hOutputReverse: 103, // DONE
+    x5hBackplaneBrightness: 104, // DONE
```

Comment:
Some of these datapoints are already defined for other devices.
Should I use predefined datapoints or create new ones for my device?
Is it possible some of those devices may receive a firmware update with updated datapoints and this will break my device?

Reply:
This is fine